components/cloud-foundry/src/view/applications/workflows/add-service-workflow/add-service-workflow.directive.js

```diff
@@ -104,13 +104,14 @@
         hideStepNavStack: true,
         lastStepCommit: false,
         btnText: {
-          cancel: config.confirm ? gettext('Cancel') : gettext('Back to Services')
+          cancel: config.confirm ? $translate.instant('buttons.cancel')
+            : $translate.instant('app-tabs.services.add.back-to-services')
         },
         steps: [
           {
             templateUrl: path + 'instance.html',
             formName: 'addInstanceForm',
-            nextBtnText: gettext('Add Service Instance'),
+            nextBtnText: $translate.instant('app-tabs.services.add.add'),
```

Comment:
Feels like this should be done via a translate in the wizard control rather than a lookup here - if so, there are other occurrences that can be changed too.